paddle/fluid/pir/dialect/operator/ir/ops.yaml

```
@@ -454,6 +454,13 @@
   optional : in_accum, in_state, out_scale, out_accum, out_state
   inplace : (scale -> out_scale, in_accum -> out_accum, in_state -> out_state)
 
+- op : dgc
+  args : (Tensor u, Tensor v, Tensor grad, Tensor param, Tensor current_step, Tensor nranks, float[] sparsity, float m=0.9, bool use_nesterov=true, float rampup_begin_step=0.0, float rampup_step=0.0, float regular_coeff=0.0, int regular_type=0)
+  output : Tensor(u_out), Tensor(v_out), Tensor(encode_grad), Tensor(grad_out), Tensor(k), Tensor(gather_buff)
+  kernel :
+    func : dgc
```

```suggestion
    func : dgc
    param: [u, v, ...]
```
在kernel里面需要指定传给dgc_kernel参数，顺序和kernel参数一致，其他地方不用修改

---

多谢，已修改，请问一下这里为什么要添加呢